tests/conftest.py

```diff
@@ -433,7 +433,8 @@ def remove_tags(key, content):
     def remove_imagerepository(key, content):
         content = json.loads(content)
         content['spec'].pop('dockerImageRepository', None)
-        content['metadata']['annotations'].pop(ANNOTATION_INSECURE_REPO)
+        if content['metadata']['annotations'].get(ANNOTATION_INSECURE_REPO):
```

Comment:
You can do just `content['metadata']['annotations'].pop(ANNOTATION_INSECURE_REPO, None)`

Reply:
Thanks!  Fixed.